app/controllers/shopping_lists_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ShoppingListsController < ApplicationController
```

Comment:
Fix shopping list unique name and number, right now it keeps adding on the shopping list without checking if anything is on the shopping list.